body.tex

```diff
@@ -0,0 +1,51 @@
+
+
+\section{Introduction}
+\subsection{Scope}
+This document covers releases of software from the Data Management Subsystem of LSST.
+It discusses the delineation between the Data Facility as an operational entity and DM producing and testing software.
+
+\section{Release management}\label{sect:relman}
+
+The release manager will be the final owner of this process the current understanding is outlined  here.
+
+\subsection{Preparation of Releases}\label{sect:relprep}
+DM develops code in github following the developer guidelines \url{https://developer.lsst.io/}.
+This includes automated testing and continuous integration. Tested releases are tagged by SQuARE
+weekly and major releases are made each Cycle (six months).
+
+There are specific packages and systems deployed together to form the high level components of DM as depicted in \figref{fig:dmsdeploy}. The orchestration of deployments on multiple machines is facilitated by the use of contains and a machine readable configurations. DM prepares docker containers and puppet configurations for deploying these systems on Kubernetes enabled cluster. These configurations are also tagged in the release.
+
+In addition specific releases with features required to support the \cite{LDM-503} test milestones will be tagged and released in advanced of each verification test. The initial feature lists for these releases are defined in \secref{sect:features}.
```

Comment:
I note "in addition". I think the wording in DMTN-020 implies (although isn't totally explicit on the point) and it was certainly my understanding that we wanted feature-based releases to _replace_ the time based release process.